src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -2173,6 +2173,30 @@ public TimeSpan SqlTracesHarvestCycle
             }
         }
 
+        private TimeSpan? _updateLoadedModulesCycleOverride = null;
+        public TimeSpan UpdateLoadedModulesCycle
```

Comment:
For consistency you might want to rename this to `UpdateLoadedModulesHarvestCycle`.

Reply:
I guess I considered this more like GetAgentCommands than a harvest cycle like for metrics.  I can rename it, but I don't think its necessarily needs it.

Reply:
One of the reasons I brought it up is because it's reported up to the server with the agent settings as a "harvest cycle".

Reply:
Oh! That’s a copy-paste fail on my part. It should not say  that.